data/presets/power/portal.json

```diff
@@ -0,0 +1,29 @@
+{
+    "icon": "temaki-power",
+    "fields": [
+        "ref",
+        "operator",
+        "structure",
+        "design",
```

Comment:
I'd move the `design` field to `moreTags`. The values for this tag require quite some domain knowledge, and are not something a typical mapper is expected to enter.